tuxemon/core/states/choice/__init__.py

```diff
@@ -16,11 +16,11 @@ class ChoiceState(PopUpMenu):
     * if there are no more messages, then the dialog will close
     """
     shrink_to_items = True
-    escape_key_exits = False
```

Comment:
minor.  please leave this because it is used by linters to find out what attributes could be used by a class/instance.  if removed and added in any other method besides `__init__` linters will complain that it is not a value defined in the class.  You could set it to None here so that the default is not specified in two places.